util/platform.zig

```diff
@@ -30,6 +56,7 @@ pub fn linux(allocator: std.mem.Allocator) !OsInfo {
     const memory = try lnx.getTotalMemory();
 
     return OsInfo{
+        .allocator = allocator,
```

Comment:
I had some other ideas around the `OsInfo` thing but moved them to another branch, this was a small change to make the structure usable as not just an internal thing. See https://github.com/bens/zBench/commit/59f8d7ae0a892bc995eba187cec4fb283310061e for the other ideas, basically I wanted to avoid needing an allocator, only use stack while fetching the info and then cache it in static memory. Then it never needs to be deinited.